src/sentry/relay/config/__init__.py

```diff
@@ -152,7 +152,14 @@ def get_filter_settings(project: Project) -> Mapping[str, Any]:
 
 
 def get_quotas(project: Project, keys: Optional[Sequence[ProjectKey]] = None) -> List[str]:
-    return [quota.to_json() for quota in quotas.get_quotas(project, keys=keys)]
+    try:
+        computed_quotas = [quota.to_json() for quota in quotas.get_quotas(project, keys=keys)]
```

Comment:
At first, I moved the `quota.to_json()` part from the `try` section to the `else` section. However, that `to_json` call is also part of the critical path for project configs, so I decided to keep it this way.